win32ss/user/ntuser/menu.c

```diff
@@ -2803,12 +2803,11 @@ static BOOL MENU_InitPopup( PWND pWndOwner, PMENU menu, UINT flags )
     if (flags & TPM_LAYOUTRTL || pWndOwner->ExStyle & WS_EX_LAYOUTRTL)
        ex_style = WS_EX_LAYOUTRTL;
```

Comment:
Slightly unrelated, but looking at line 2797 this should be "|=" instead of "=" so it will not override the previous flags set.